### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by psypherium**
- Add recompilation instructions
- Add build threading instructions
- Rearranged 1 sentence with passive voice to match the rest of the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,6 +80,19 @@ make -j4
 ```
 STK can then be run from the build directory with `bin/supertuxkart`
 
+### Recompiling
+
+To recompile the latest code without redownloading the entire source, first run the ```svn up``` command inside the 'stk-assets' directory, then run the following commands inside the 'stk-code' directory:
+
+```git pull
+make clean
```
Comment (auriamg) [inline README.md]: Make clean shouldn't be required, also there is no makefile at the root so that wouldn't work?

Comment (psypherium) [reply README.md]: Ah, yes that makes sense. I must be getting that part confused with the 0ad rebuild process. I'll rebuild again after the next commit and make sure that make clean is definitely not required.

Comment (psypherium) [reply README.md]: The commands worked perfectly without make clean.

**Comment by psypherium** [comment]
- Removed unnecessary "make clean" command

**Comment by deveee** [comment]
Tbh. I don't see any reason why we should add information about recompilation.

**Comment by psypherium** [comment]
"To recompile the latest code without redownloading the entire source"

The word recompilation is not the right one. the 0ad build instructions say "keeping up to date", i guess I just wanted to be more fancy.

The build threading instructions are for building the code faster.

The sentence rearrangement was for readability as every other sentence was written in the passive voice.

**Comment by auriamg** [comment]
Agreed that "keeping up to date" sounds better, "recompile" seems to hint "from scratch" which is not the case

**Comment by auriamg** [comment]
Thanks, merged

**Pull request merged**